test/evergreen.yml

```diff
@@ -2853,6 +2902,23 @@ buildvariants:
   - name: ".stress-test-ppc-1"
   - name: ".stress-test-ppc-2"
 
+- name: ubuntu1804-ppc-cmake
+  display_name: "~ Ubuntu 18.04 PPC CMake"
+  run_on:
+  - ubuntu1804-power8-test
+  expansions:
+    test_env_vars: LD_LIBRARY_PATH=$(pwd)
+    posix_configure_flags: -DHAVE_DIAGNOSTIC=1 -DENABLE_PYTHON=1 -DENABLE_ZLIB=1  -DENABLE_SNAPPY=1 -DENABLE_STRICT=1 -DCMAKE_INSTALL_PREFIX=$(pwd)/LOCAL_INSTALL
+    python_binary: '/opt/mongodbtoolchain/v3/bin/python3'
+    smp_command: -j $(grep -c ^processor /proc/cpuinfo)
+    cmake_generator: Ninja
+    make_command: ninja
+    is_cmake_build: true
+  tasks:
+    - name: compile
+    - name: make-check-test
+    - name: ".unit_test"
+
```

Comment:
I'm not sure we'd like to introduce new builders/tasks for PPC & zSeries, considering the building resources are really limited atm - don't hope to overload the 2 distro pools and slow down the make span of corresponding authconf builders further. I think we need some tradeoff of testing efficiency over coverage here.

Reply:
Thanks for the context. I'm slightly worried the builds might regress since there were nuances I had to debug when implementing CMake support on these architectures. Is there a way to keep the variants but to schedule them less frequently? (e.g. don't run by default on Waterfall regression testing).
If not, I'm ok to remove the PPC and ZSeries CMake variants. Given those variants are on Ubuntu, we should be able to rely on the x86 Ubuntu variant to test CMake (as they _mostly_ have the same system headers).

Reply:
Thats fair. Yes, we do have a parameter in Evergreen to control the scheduling frequency - ` batchtime`, the value is in mins so e.g. 1440 equals 1 day. You should be able to find some examples in the evergreen.yml file.

Reply:
Awesome! Thanks for pointing me to `batchtime`, I wasn't aware of the feature. I've updated the PPC & ZSeries variants to run every 7 days (same frequency as the `big-endian` variant).